lms/static/styles/_variables.scss

```diff
@@ -34,6 +34,16 @@ $subtitle-font-size: 15px;
 
 $input-font-size: 15px;
 
+// Layout spacing
+// -------------------------
+$layout-space: 1em;
```

Comment:
We don't really need these yet, but I thought it might be good to start copies over the common vars